### PR TITLE
Implement SystemInformationApi for the GET endpoint /api/rest/v1/system-information

### PR DESCRIPTION
**Pull request by ConstantBqt**
Get the data from this endpoint https://api.akeneo.com/api-reference.html#get_system_information 

Available in PIM versions: 6.0, 7.0 and SaaS

### CODE REVIEW

**Comment by ConstantBqt** [comment]
What are the next steps to get this PR approved ?

**Comment by ConstantBqt** [comment]
Hello @tomglvng, 
I saw your name as reviewer for another PR of this repository. Could you please review this PR when you get a chance ? 
Thanks a lot !

**Comment by tomglvng** [comment]
> Hello @tomglvng, I saw your name as reviewer for another PR of this repository. Could you please review this PR when you get a chance ? Thanks a lot!

*Hello @ConstantBqt ,*

*Thank you so much for your contribution! To continue the merge process, we would need you to sign our Contributor License Agreement kindly. Please find it here: https://www.akeneo.com/contributor-license-agreement/*

*Looking forward to reading you,*

*The Akeneo development team*

File: src/Api/SystemInformationApi.php
Comment (tomglvng) [inline src/Api/SystemInformationApi.php:1]: You should add tests for this class. Get inspiration with spec/Api/AttributeGroupApiSpec.php

**Review by tomglvng — CHANGES_REQUESTED** [review]
Did you launch tests?
make tests

File: src/Api/SystemInformationApi.php
Comment (ConstantBqt) [reply src/Api/SystemInformationApi.php:1]: Hi @tomglvng 
I added `SystemInformationApiSpec` and fix the `AkeneoPimClientSpec` in my last commit. 
The result of `make tests` is successful 

Thanks 🙏

**Comment by lasdou** [comment]
Hello, Any updates regarding this PR @tomglvng ? I would be interested by this new endpoint too. Thanks ;-)

**Comment by lasdou** [comment]
CI is green so merging it should be ok. Waiting for @tomglvng review

**Comment by tomglvng** [comment]
https://github.com/akeneo/api-php-client/pull/292

**Comment by tomglvng** [comment]
Hello @ConstantBqt , SOrry for the delay. I created a PR to merge your contribution. I'm waiting for the review. Could you have a look @lasdou ?

**Comment by tomglvng** [comment]
@ConstantBqt merged! Thanks for your contribution ;)

**Pull request closed without merging**